README.md

```diff
@@ -19,6 +19,39 @@ inject type information into your interaction handlers.
 >
 > The `discord_app` library is currently only known to be available in Deno.
 
+### Development
+
+When developing with the `discord_app` library, make sure to include all of the 
+necessary environmental variables from the Discord application into its designated 
+`.env` file.
```

Comment:
It may be helpful to the reader to see a list of which secrets belong in the `.env` file.

Reply:
Added necessary environmental variables in!